activerecord-cockroachdb-adapter.gemspec

```diff
@@ -1,10 +1,16 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+
+lib = File.expand_path('lib', __dir__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
+version_line = File.open('./lib/active_record/connection_adapters/cockroachdb/version.rb') do |f|
+  f.find {|line| line.include?("VERSION")}
+end
+version = version_line.split('=').last.strip.gsub("\"", '')
+
```

Comment:
Ideally, we'd be able to require the module defined in `version.rb` or `load` it, but doing either of those breaks the test runner. I tried to look into it, but couldn't figure out why loading that module caused issues. Basically, all the subdirectories of `active_record` were not loaded in the case we tried to load in the version.

Assuming the structure of the file version file doesn't change, this workaround should suffice, but we could try to investigate it more later if we think it'll be an issue.

Reply:
We could also hardcode it and remember to change it in both places.

Reply:
For whatever reason including the version under the `ConnectionAdapters` module caused loading issues, so I just moved the constant to `ActiveRecord::CockroachDBAdapterVersion` and that fixed the issue.